### DISCUSSION TITLE
Projects (Beta): Project progress

### DISCUSSION THREAD

**Discussion by MichielDeMey** [community/community#5523] [Projects and Issues]
The old projects had the ability to show the total progress of the project.
This seems to be absent from the new Projects (beta).

Is this something that will come back?

<img width="857" alt="Screenshot 2021-09-08 at 14 44 23" src="https://user-images.githubusercontent.com/793406/132511440-98cdbd15-bf96-4488-85c6-a2599c66acb5.png">

**Comment by connor-torrey** [comment]
I liked that feature, but it'd be nice to be able to add some 'effort' to each task that was reflected in the progress bar.  When each card is one 'unit', sometimes it appears as if the project is almost done, when in reality only a bunch of small tasks have been completed.  You could even have an 'Estimated Effort' and 'Applied Effort' in a card to track progress within the issue
